test/CoveragePool.test.js

```diff
@@ -121,9 +121,22 @@ describe("CoveragePool", () => {
       })
     })
 
-    context("when caller is the governance", () => {
+    context("when first risk manager was not approved", () => {
+      it("should revert", async () => {
+        await expect(
+          coveragePool
+            .connect(governance)
+            .beginRiskManagerApproval(riskManager.address)
+        ).to.be.revertedWith("First risk manager was not approved")
+      })
+    })
+
+    context("when first risk manager was approved", () => {
```

Comment:
For completeness, we can say: "when called by the governance and first risk manager was approved"

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/176/commits/3aa4ecce94e64166f1f4fece8d634156dab0ef1c.